Source/Blazorise/Attributes/SelectAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Blazorise;
+
+/// <summary>
+/// Represents an attribute that can be applied to properties or fields to specify additional metadata for a select component.
+/// </summary>
+[AttributeUsage( AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false )]
+public sealed class SelectAttribute : Attribute
+{
+    /// <summary>
+    /// The name of the function that will be used to get the data source.
+    /// By default it's set to "GetSelectData".
+    /// </summary>
+    public string GetDataFunction { get; set; } = "GetSelectData";
+
+    /// <summary>
+    /// Used to get the display field from the supplied data source.
+    /// Defaults to "Description".
+    /// </summary>
+    public string TextField { get; set; } = "Description";
```

Comment:
Should we even have default names for the fields? The combination of `Description` and `Id` seems very rare in the real world.

Maybe name them differently?

- `Value`
- `Text`

Reply:
Well rare for you...? :D I'm thinking in the perspective of someone binding a DTO.
So at least, I have worked with Id commonly. So wanting the **Id** as the **Value** which corresponds to the database entity. That seems like a common case? Even if its a DTO the dto generally also points to the Id. 

As for the **Text** I think it's generally Description or Text,..?

Reply:
I also don't mind not assuming any defaults.
The idea was that if someone were to have the same type of binding, he would have to define less metadata in the attribute.

Reply:
Here's an example on our own code base, for support forum, that if we used a `SelectList` the `ValueField` would be the `Id`
![image](https://github.com/Megabit/Blazorise/assets/22283161/4401c36a-be8f-4deb-80d3-1b392dced3d3)

Reply:
Done.